### PR TITLE
Added open command

### PR DESCRIPTION
**Pull request by Likey00**
The open command pulls from the SOC api to check whether a given section index is currently open or closed

Usage: !open "section index"

### CODE REVIEW

**Comment by rosalogia** [comment]
Would it make sense to ask that this functionality at least be contained in a separate file? I think that would be more organised. We don't need to use cogs if you don't want to, but some file separation would be good.

EDIT: That being said, the code looks good

File: main.py [hunk 1]
```diff
@@ -18,4 +20,30 @@ async def echo(ctx, *args):
...
+    open_sections_url = "https://sis.rutgers.edu/soc/api/openSections.gzip?year=2020&term=9&campus=NB"
+
+    #Try catch in case the api call fails
+    try:
+        #I load the text from requests call into a list called open_sections
+        #open_sections now contains indexes of all open sections
+        open_sections = json.loads(requests.get(open_sections_url).text)
+    except:
+        #If something went wrong, output API error and exit function
+        await ctx.send("API Error, sorry!")
```
Comment (rosalogia) [inline main.py]: Maybe we should log the actual error as well. We want to be able to debug if there's a legit issue that isn't just bad network.

**Comment by rosalogia** [comment]
Do we still need test.py? Would it make sense for you to keep that but gitignore it maybe?

**Comment by Likey00** [comment]
Yea meant to delete that, let me fix it

**Comment by Likey00** [comment]
The bot now uses cogs :)

**Review by rosalogia — APPROVED** [review]

**Pull request merged**